arbnode/node.go

```diff
@@ -1425,6 +1424,7 @@ func (n *Node) Start(ctx context.Context) error {
 	if n.configFetcher != nil {
 		n.configFetcher.Start(ctx)
 	}
+	n.SyncMonitor.Initialize(n.InboxReader, n.TxStreamer, n.SeqCoordinator)
```

Comment:
nice catch!
please add a comment here explaining the order. These are things we don't want to have to relearn.

Reply:
Done.